ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+## Please remove this line and fill the template
```

Comment:
Should these be `<!--  -->` style comments (as in https://raw.githubusercontent.com/nodejs/node/master/.github/ISSUE_TEMPLATE.md)?